B/Blosc2/build_tarballs.jl

```diff
@@ -62,9 +63,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Lz4_jll"; compat="1.9.3"),
-    Dependency("Zlib_jll"),
-    Dependency("Zstd_jll"; compat="1.5.0"),
+    Dependency("Lz4_jll"; compat="1.10.0"),
+    Dependency("Zlib_jll"; compat="1.3.1"),
```

Comment:
This will work only on Julia master, not very useful for most users for several months

Reply:
How do I choose a compat bound that's good for aarch64-unknown-freebsd as well? Leave it out? Make it depend on the Julia version?

Reply:
Pkg doesn't allow platform/version-depending compat bounds (the project is unique across all systems), which leaves you only with the former option of leaving this for the time being.